Fix:
pkcs11/xdg-store/mock-xdg-module.c:91:2: error: implicit declaration of function 'gettimeofday' is invalid in C99 [-Werror,-Wimplicit-function-declaration]
pkcs11/xdg-store/mock-xdg-module.c:95:6: error: implicit declaration of function 'utimes' is invalid in C99 [-Werror,-Wimplicit-function-declaration]
https://gitlab.gnome.org/GNOME/gnome-keyring/-/merge_requests/48
--- pkcs11/xdg-store/mock-xdg-module.c.orig	2018-06-24 23:15:03.000000000 -0500
+++ pkcs11/xdg-store/mock-xdg-module.c	2022-03-23 21:25:58.000000000 -0500
@@ -36,7 +36,7 @@
 #include <glib/gstdio.h>
 
 #include <errno.h>
-#include <sys/times.h>
+#include <sys/time.h>
 
 #include <string.h>
 
